### DISCUSSION TITLE
SQLite shows unable to open default font

### DISCUSSION THREAD

**Discussion by nataliegao1** [sqlitebrowser/sqlitebrowser#3628] [General]
After installing SQLite and opening it shows unable to open default font, followed by the path, have tried to restore the font settings, the sqlite version is also compatible with the computer. querying SQLite3 in CMD shows that it was not successfully installed, but I already installed SQLite and can open it.
![image](https://github.com/sqlitebrowser/sqlitebrowser/assets/130290858/d72c862b-c3fa-40ea-b8a8-32f4b5aa1338)

**Comment by chrisjlocke** [comment]
Not an overly helpful message.  
Deleting the EUDC registry key (and the sub-keys) from the registry is the quick fix.
See this issue: https://github.com/sqlitebrowser/sqlitebrowser/issues/2556

Yell back if it works / doesn't work.
